docs/latest/design/roadmap.md

```diff
@@ -32,13 +32,18 @@ contributing to the project.
 - Achieve Gateway API conformance (e.g. routing, LB, Header transformation, etc.)- [Issue #65][65].
 - Setup a CI/CD pipeline- [Issue #63][63].
 
-### [v0.3.0][v0.3.0]: Drive Advanced Features through Extension Mechanisms
+### [v0.3.0][v0.3.0]: Driving Advanced Features through Extension Mechanisms
 
-- Global Rate Limiting
-- AuthN/AuthZ- [Issue #336][336].
-- Lets Encrypt Integration
+- Support extended Gateway API fields.
+- Support experimental Gateway APIs such as TCPRoute [Issue #643][643], UDPRoute [Issue #641][641] and GRPCRoute [Issue #642][642].
+- Rate Limiting [Issue #670][670].
+- Authentication [Issue #336][336].
 
-### [v0.4.0][v0.4.0]: Manageability and Scale
+### [v0.4.0][v0.4.0]: More Advanced Features through Extension Mechanisms
+
+- Allow users to configure xDS Resources [Issue #24][24].
```

Comment:
I suggest moving some of the v0.3 deliverables to v0.4. Note that the original v0.3 deliverables were based on a 6-month release cycle.

Reply:
This is my concern too.